File: integration-tests/gradle/src/main/resources/multi-module-included-build/app/src/main/java/org/acme/ExampleResource.java
package org.acme;

import org.acme.lib.LibService;

import jakarta.inject.Inject;
import jakarta.ws.rs.GET;
import jakarta.ws.rs.Path;
import jakarta.ws.rs.Produces;
import jakarta.ws.rs.core.MediaType;

@Path("/hello")
public class ExampleResource {

    @Inject
    LibService libService;

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    public String hello() {
        return "foo " + libService.bar();
    }
}

File: integration-tests/gradle/src/main/resources/included-kotlin-kmp-build/src/main/kotlin/org/example/ExampleResource.kt
package org.example

import jakarta.ws.rs.GET
import jakarta.ws.rs.Path
import jakarta.ws.rs.Produces
import jakarta.ws.rs.core.MediaType

@Path("/hello")
class ExampleResource() {

    @GET
    @Path("/kmp")
    @Produces(MediaType.TEXT_PLAIN)
    fun helloKmp() = SomeObject.aString

    @GET
    @Path("/jvm")
    @Produces(MediaType.TEXT_PLAIN)
    fun helloJvm() = AnotherObject.aString
}